build.sbt

```diff
@@ -12,7 +12,7 @@ import laika.config.LinkValidation
 import org.typelevel.sbt.site.TypelevelSiteSettings
 import sbt.librarymanagement.Configurations.ScalaDocTool
 // https://typelevel.org/sbt-typelevel/faq.html#what-is-a-base-version-anyway
-ThisBuild / tlBaseVersion := "0.10" // your current series x.y
+ThisBuild / tlBaseVersion := "0.11" // your current series x.y
```

Comment:
Mima [atempted](https://github.com/typelevel/weaver-test/actions/runs/17401344245/job/49394996853#step:11:24) to fetch `_native0.5` variants, but `0.10` was published against 0.4 causing compatibility check failures

```
[error] sbt.librarymanagement.ResolveException: Error downloading org.typelevel:weaver-discipline_native0.5_2.13:0.10.0
[error]   Not found
[error]   Not found
[error]   not found: /home/runner/.ivy2/localorg.typelevel/weaver-discipline_native0.5_2.13/0.10.0/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/typelevel/weaver-discipline_native0.5_2.13/0.10.0/weaver-discipline_native0.5_2.13-0.10.0.pom
```

I suppose the best way out of this is to go 0.11.x